design/research/2026-06-08-humanizing-ai-prose/web-research.md

```
@@ -0,0 +1,223 @@
+## Sources Found
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)